features/facebookImport/src/model/entities/facebook-account.js

```diff
@@ -243,6 +244,14 @@ class FacebookAccount {
         this._postReactions = postReactions;
     }
 
+    get posts() {
+        return this._posts;
+    }
+
+    set posts(posts) {
```

Comment:
Are we going to change this once the instance has been created?

Reply:
Actually not. Now we push into the array so I guess we can remove the setter.